apispec-model/src/main/scala/sttp/apispec/Schema.scala

```diff
@@ -2,14 +2,30 @@ package sttp.apispec
 
 import scala.collection.immutable.ListMap
 
-// todo: discriminator, xml, json-schema properties
+sealed trait SchemaLike
+
+sealed trait AnySchema extends SchemaLike
+
+object AnySchema {
```

Comment:
suggestions for names here?

Reply:
not entirely sure if its useful to have the encodings in the model, or if this should possibly be a configuration of encoder.